packages/core/datadog-sourcemaps.gradle

```
@@ -39,55 +41,45 @@ afterEvaluate {
             logger.info("Cannot find JS bundle task for variant=${targetName}.")
             return
         }
+
         if (!bundleTask.enabled) {
             logger.info("JS bundle task for variant=${targetName} is not enabled.")
             return
         }
 
+        def sourcemapOutput
+        def bundleOutput
+
+        (bundleOutput, sourcemapOutput) = forceSourcemapsGenFromBundleTask(bundleTask)
```

I think it should be possible to merge declaration and assignment:

```suggestion
        def (bundleOutput, sourcemapOutput) = forceSourcemapsGenFromBundleTask(bundleTask)
```